flang/include/flang/Lower/Runtime.h

```diff
@@ -20,6 +20,8 @@
 #ifndef FORTRAN_LOWER_RUNTIME_H
 #define FORTRAN_LOWER_RUNTIME_H
 
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
```

Comment:
If you are adding this here only for the mlir::Value, it is not wrong, but you could also add a `class Value;` pre-declaration in the `namespace mlir {` below.